src/ocm/components/clusterConfiguration/DiscoveryImageSummary.tsx

```diff
@@ -11,6 +12,16 @@ type DiscoveryImageSummaryProps = {
 
 const DiscoveryImageSummary: React.FC<DiscoveryImageSummaryProps> = ({ cluster, ...restProps }) => {
   const { imageUrl, error, isLoading } = useInfraEnvImageUrl(cluster.id);
+  const [fileName, setFileName] = useState('discovery.iso');
+
+  useEffect(() => {
+    async function setInfraEnvId() {
+      const infraEnvId = await InfraEnvsService.getInfraEnvId(cluster.id);
+      setFileName(`${infraEnvId}-discovery.iso`);
+    }
+    setInfraEnvId();
+  }, [cluster.id]);
+
```

Comment:
We should/can get the file name from the content-disposition header
![image](https://user-images.githubusercontent.com/77008341/159340405-72a40c0e-5a30-49d1-b0f3-0c0373a8c427.png)

Reply:
If in the future they change the file name format we won't need to do anything...

Reply:
Right now the HEAD request is blocked by CORS (probably the same will happen in production).
I would recommend asking the BE team to add the CORS headers for this endpoint first...

Reply:
@jkilzi But this is not for the direct download but for the "wget ..." field. I don't want to download the file in the background just to get its name for the "wget ..." field and then again if the user chooses to press the "Download Discovery ISO" button.

Reply:
@osherdp FYI

Reply:
@jgyselov what do you mean about downloading the file in the background?
HEAD requests don't download the content itself

as for the CORS, I remember that we couldn't find any evidence for a CORS problems in production
it might still exist for staging though, so if you see anything please open a bug

Reply:
Nick's PR is merged we should be able now to call make the API call with the HEAD method